app/flows/check_travel_during_coronavirus_flow/start.erb

```diff
@@ -44,4 +44,13 @@
   <p>If you’re travelling to England from within the Common Travel Area (England, Scotland, Wales and Northern Ireland, Ireland, the Channel Islands and the Isle of Man) and haven’t been anywhere else within the 10 days before you arrive, there are no entry requirements for coming into England.</p>
 
   <p>If you’ve been to a country or territory outside the Common Travel Area within the 10 days before you arrive in England, you must follow the rules for entering England from that country. You can use this tool to find out those rules.</p>
+
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "Travelling to England from outside the Common Travel Area",
+    heading_level: 2,
```

Comment:
I think this heading should be smaller? Maybe a level 3?

Reply:
Changed to level 3 - thanks @leenagupte  - I missed that in the Google doc - they looked the same size 👍